lib/validations.js

```diff
@@ -88,6 +89,17 @@ validations.isAlphanumeric = function isAlphanumeric(paramName, customMessage) {
   );
 }
 
+validations.isArray = function isPlainObject(paramName, customMessage) {
+  return function(params) {
+
+    return {
+      field: paramName,
+      message: customMessage || message(paramName, "must be an array"),
+      result: Array.isArray(get(params, paramName))
```

Comment:
We could probably get away with using `isArray` from lodash here, to keep things consistent across different js environments.